dap_aria_mapping/getters/openalex.py

```diff
@@ -78,6 +78,27 @@ def get_openalex_entities() -> Mapping[str, Mapping[str, Union[str, str]]]:
         download_as="dict",
     )
 
+def get_openalex_topics(tax: str = 'cooccur', level: int = 1) -> Dict:
```

Comment:
At some point we should pin down the labelling of taxonomies, as you've named your taxonomy folders "community_detection" but this still assumes the "cooccur" label. The simulation code is now using "community" as the label for the taxonomy so, while not an issue, we're introducing naming conflicts in the latest branches.